packages/components/psammead-section-label/src/index.jsx

```diff
@@ -71,8 +71,12 @@ const SectionLabel = ({
   script,
   service,
   visuallyHidden,
+  sectionName,
```

Comment:
we already pass a prop for `labelId` and it looks like it's literally just the name of the section. Could that be used instead of passing a new prop?

Reply:
Yeah, I didn't see that